### PR TITLE
Add vuln-type parameter

### PR DESCRIPTION
**Pull request by chr4**
This adds an argument for the `vuln-type` parameter.

### CODE REVIEW

**Comment by anandg112** [comment]
👍

File: action.yaml [hunk 1]
```diff
@@ -13,6 +13,10 @@ inputs:
     description: 'ignore unfixed vulnerabilities'
     required: false
     default: false
+  vuln-type:
+    description: 'list of vulnerability types (os,library)'
```
Comment (simar7) [inline action.yaml:17]: Could you change this to say ` comma-separated list of vulnerability types (os,library)` instead?

**Review by simar7 — CHANGES_REQUESTED** [review]
thanks for the PR! – I just left a comment for a change.

File: action.yaml (hunk 1 above)
Comment (chr4) [reply action.yaml]: Done!

**Review by simar7 — APPROVED** [review]

**Pull request merged**